doc/adrs/128_add_support_for_confirmation_dialogs.adoc

```diff
@@ -0,0 +1,53 @@
+= ADR-128 - Add support for confirmation dialogs
+
+== Context
+
+Sirius Web does not have an undo/redo mechanism.
+When an element has been deleted, either from the Explorer view, from any representation, or any other view, there is no way to retrieve it.
+All delete actions are executable in one click.
+Any accidental click on a delete action can lead to the unwanted loss of data.
+By extension, some one-click actions may lead to unwanted consequences.
+
+== Decision
+
+The first decision is about the delete actions (click on the delete tool on a node, click on DEL key on a node, click on the delete menu item on an object in the Explorer view).
+For these actions, a confirmation dialog will be added, asking to confirm the deletion.
+It will be possible to disable these dialogs by clicking on the "Don't ask me again" checkbox inside them.
+These checkboxes will be associated to cookies, allowing to remember which dialogs have been disabled or not, for each end-users.
```

Comment:
Maybe use local storage instead of cookies? Cookies are useful when we need to send back some id/data along with a backend request, but here the decision to show the decision to show the dialog or not is purely frontend.

Reply:
I agree, local storage would indeed be better 👍